packages/dev/core/src/Materials/GreasedLine/greasedLinePluginMaterial.ts

```diff
@@ -50,6 +51,12 @@ export class MaterialGreasedLineDefines extends MaterialDefines {
      */
     // eslint-disable-next-line @typescript-eslint/naming-convention
     GREASED_LINE_CAMERA_FACING = true;
+
+    /**
+     * True if the line is in camera facing mode

```

Comment:
The doc is a copy/paste from `GREASED_LINE_CAMERA_FACING`.

Reply:
Fixed.